assignment2.yaml

```diff
@@ -0,0 +1,150 @@
+swagger: "2.0"
+info:
+  description: "Welcome to the swapi, the Star Wars API! This documentation should help you familiarise yourself with the resources available and how to consume them with HTTP requests. If you're after a native helper library then I suggest you scroll down and check out what's available. Read through the getting started section before you dive in. Most of your problems should be solved just by reading through it."
+  version: "1.0.0"
+  title: "SWAPI"
+host: "swapi.co"
+basePath: "/api"
+tags:
+- name: "Films"
+  description: "A Film resource is a single film."
+  externalDocs:
+    description: "Find out more"
+    url: "http://swapi.co/api/films/1/"
+schemes:
+- "http"
+paths:
+  /films/:
+    get:
+      tags:
+      - "Films"
+      summary: "get all the film resources"
+      consumes:
+      - "application/json"
+      produces:
+      - "application/json"
+      parameters:
+      - in: "body"
+        name: "body"
+        description: "Schema information for returned response"
+        required: true
+        schema:
+          $ref: "#/definitions/Films"
+      - name: "title"
+        in: "query"
+        description: "Search field"
+        required: true
+        type: "string"
+        collectionFormat: "multi"
+      responses:
+        200:
+          description: "OK"
+  /films/:id/:
+    get:
+      tags:
+      - "Films"
+      summary: "get a specific film resource"
+      consumes:
+      - "application/json"
+      produces:
+      - "application/json"
+      parameters:
+      - in: "body"
+        name: "body"
+        description: "Schema information for returned response"
+        required: true
+        schema:
+          $ref: "#/definitions/Films"
+      responses:
+        200:
+          description: "OK"
+  /films/schema/:
+    get:
+      tags:
+      - "Films"
+      summary: "view the JSON schema for this resource"
+      consumes:
+      - "application/json"
+      produces:
+      - "application/json"
+      parameters:
+      - in: "body"
```

Comment:
seriously!!! is this the body parameter. I guess we will be doing GET call here !!!